src/libclipper/src/redis.cpp

```diff
@@ -209,6 +205,24 @@ boost::optional<std::string> get_current_model_version(
   return boost::none;
 }
 
+std::vector<std::string> get_linked_models(redox::Redox& redis,
+                                           const std::string& app_name) {
+  std::vector<std::string> linked_models;
+  if (send_cmd_no_reply<string>(
+          redis, {"SELECT", std::to_string(REDIS_APP_MODEL_LINKS_DB_NUM)})) {
+    auto result =
+        send_cmd_with_reply<std::vector<string>>(redis, {"SMEMBERS", app_name});
+    linked_models = *result;
```

Comment:
We should check that `result` is present before attempting to dereference it from the `optional` container.